### DISCUSSION TITLE
What could be reasons to miss a detection

### DISCUSSION THREAD

**Discussion by hua-bing** [blakeblackshear/frigate#14869] [Ask A Question]
I have a working 0.12.1 six-HD camera setup using openvino as the detector to detect person, cat, and bird.  The detection dimension is 1920 X 1080.  I also set up to have 24 hr continuous recordings from all my cameras.  My system is running on a QNAP NAS with a J3455 CPU (4-core) and 16G of RAM.  I assigned 50% CPU and 4G of RAM to Frigate docker container.  The shm-size is 740 M (should be more than enough, only shown as 4% is being used as I just checked).

While the system has been running for a couple of years, I just noticed recently that there are missing events from at least one of my cameras.  At first, I thought these are probably occasional glitches.  But then I realized that it might actually happen pretty often.  For example, this morning I walked pass this camera.  But after I came back to check, I see no event is reported.  I checked the recordings and verified that the event is present in the 24 hr recording of the camera, ruling out the possibility of the camera was not working at that time.  I do not see anything in the log regarding to this camera during that period of time.

The particular camera is installed as a top-down view, meaning that at the extreme angle, it sees mostly the top of person's head and shoulders.  However, I do see events were reported for successful detections from this angle in the history, meaning that we can probably rule out the possibility of the model being not suitable for this particular view angle.  The camera is installed over a path that there are always persons passing by during certain time frames in the morning and afternoon.

**Comment by NickM-27** [comment]
most likely the object is not detected with enough confidence to create an event.

**Reply by hua-bing** [reply]
> most likely the object is not detected with enough confidence to create an event.

Thanks Nick.  This is a good point.  However, as people walking further, the angle is expected to be better for a higher confidence?  It's a Wyze Cam V3, with a 130-degree field view angle.  I checked the history, the lowest confidence is 70%.  However, the ballpark detection confidence is above 80%, with quite many in 90% range.  Of course there might be a "survival bias" here.

Is there a way to test the hypothesis by turning on some log settings so that all detection event is logged somewhere?

**Reply by NickM-27** [reply]
there is no log for that as there would be too much information to be usable. You can always try lowering the threshold for person

**Reply by hua-bing** [reply]
Lowering the threshold is a good idea.  Thanks a lot.
